src/style.css

```
@@ -0,0 +1,94 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row-reverse;
+  width: fit-content;
```

background-image: url(images/star.svg);
  background-size: cover;
```suggestion
  width: fit-content;
  background-image: url(images/star.svg);
  background-size: cover;
  background-position: center;
  background-repeat: no-repeat;
```